package.json

```diff
@@ -22,7 +22,7 @@
     "components:build-cdn:versions-list": "node tasks/update-versions-list.js",
     "components:publish-cdn": "node tasks/publish-cdn.js",
     "components:check-i18n": "node tasks/check-i18n.js",
-    "components:check-lit": "lit-analyzer src",
+    "components:check-lit": "lit-analyzer --rules.no-unknown-attribute off src",
```

Comment:
This should be configured in the `tsconfig.json` at the root of the projet.
But even if it's a bit weird, I'd prefer to use the `globalAttributes` config, see https://github.com/runem/lit-analyzer/blob/master/docs/readme/config-table.md

I tried this and it works :scream: 

```json
{
   ...
  "globalAttributes": [
    "________________________________",
    "___________________________________________________________"
  ],
  ...
}

Reply:
haha! nice hack!